oxide-auth-poem/src/error.rs

```diff
@@ -0,0 +1,11 @@
+use thiserror::Error;
+
+#[derive(Clone, Debug, Error)]
+pub enum OxidePoemError {
+    #[error("There was a problem with the request")]
```

Comment:
This _could_ be more specific. Something along the lines of 'Invalid request query' and 'Invalid request body' with more precise error information would also work.

Reply:
Yep, will work on that.